cmd/actions/types/flags.go

```diff
@@ -0,0 +1,18 @@
+package types
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var (
+	FlagRPC      string
+	FlagGRPC     string
+	FlagInsecure bool
+)
+
+// AddNodeFlagsToCmd adds node flags to hasura actions.
+func AddNodeFlagsToCmd(cmd *cobra.Command) {
+	cmd.PersistentFlags().StringVar(&FlagRPC, "rpc", "https://localhost:26657", "<host>:<port> to the RPC interface for the chain")
+	cmd.PersistentFlags().StringVar(&FlagGRPC, "grpc", "https://localhost:9090", "<host>:<port> to the gRPC interface for the chain")
+	cmd.PersistentFlags().BoolVar(&FlagInsecure, "insecure", false, "insecure or secure connection")
```

Comment:
```suggestion
	cmd.PersistentFlags().BoolVar(&FlagInsecure, "insecure", true, "Allow insecure connections")
```

Reply:
hihi @MonikaCat , the Bool should default to false because _--insecure_ can't take argument, it just turns the value to true on its own.
```
// secure connections: 
$ bdjuno hasura-actions

// insecure connections(FlagInsecure becomes true with --insecure):
$ bdjuno hasura-actions --insecure
```

If default to true the FlagInsecure will always be true because we can't do _"--insecure false"_

Reply:
@huichiaotsou We're using default ports `http://localhost:26657` and `http://localhost:9090` that are not secure so the flag should be set to true. You can still pass `--insecure true` flag if needed but by default the value should be set to true.

Reply:
ah okay, because --insecure actually don't take any argument, it simply turn that value to true, I'll change the flag to --secure and default to false